libvips/conversion/composite.cpp

```diff
@@ -899,7 +899,7 @@ vips_composite_base_blend3(VipsCompositeSequence *seq,
 			/* You can't sqrt a vector, so we must loop.
 			 */
 			for (int b = 0; b < 3; b++) {
-				double g;
+				float g;
```

Comment:
ditto, we could use `g = sqrtf(B[b]);` below.

Reply:
My understanding of C++ is that `sqrt` is overloaded for both float and double precision so I think we can get away with it here.

Reply:
You're right; this isn't an issue in C++. :+1: